doc/classes/VisualServer.xml

```diff
@@ -45,6 +45,14 @@
 				Once finished with your RID, you will want to free the RID using the VisualServer's [method free_rid] static method.
 			</description>
 		</method>
+		<method name="camera_reset_physics_interpolation">
+			<return type="void" />
+			<argument index="0" name="camera" type="RID" />
+			<description>
+				Prevents [code]physics interpolation[/code] for the current physics tick.
```

Comment:
It's a bit weird to me to use code markup for "physics interpolation", it should be fine as is without specific markup IMO. But if it's something that's done consistent in other similar methods it's not blocking for this PR.

Reply:
Can change this, no problem. :slightly_smiling_face:  I wasn't aware about the localization factor for using code tags. :+1: